content/docs/guides/monitoring.md

```diff
@@ -128,4 +128,39 @@ details on the dashboard.
 
 Stackdriver by default provides container level CPU/memory metrics.
 We can also define custom Prometheus metrics and view them on the Stackdriver dashboard.
-See [here](https://cloud.google.com/monitoring/kubernetes-engine/prometheus) for more detail.
+See for more [detail](https://cloud.google.com/monitoring/kubernetes-engine/prometheus).
+
+## Prometheus
+
+### Kubeflow Prometheus component
+Kubeflow provides a Prometheus [component](https://github.com/kubeflow/kubeflow/blob/master/kubeflow/core/prometheus.libsonnet).
+To deploy the Prometheus component:
+
+```
+ks generate prometheus prom --projectId=YOUR_PROJECT --clusterName=YOUR_CLUSTER --zone=ZONE
+ks apply YOUR_ENV -c prom
+```
+
+The prometheus server will scrape the services with annotation `prometheus.io/scrape=true`.
+See for more [detail](https://github.com/kubeflow/kubeflow/blob/master/kubeflow/core/prometheus.yml#L75) 
+and an [example](https://github.com/kubeflow/kubeflow/blob/master/kubeflow/core/metric-collector.libsonnet#L83).
+
+#### Export metrics to Stackdriver
+The Prometheus server will export metrics to Stackdriver, as
+[configured](https://github.com/kubeflow/kubeflow/blob/master/kubeflow/core/prometheus.yml#L127).
+We are using an [image](https://github.com/kubeflow/kubeflow/blob/master/kubeflow/core/prometheus.libsonnet#L170)
```

Comment:
Lunkai can you clarify the instructions here. If I understand the discussion with Kunming the prometheus component saves users the trouble of having to follow the GKE docs. Can we clarify that this is what the prometheus component is doing and if they use it they don't need to follow the instructions in the GKE docs.